notifier/src/main/kotlin/modules/JiraNotifier.kt

```diff
@@ -40,6 +40,16 @@ class JiraNotifier(
     private val restClient: JiraRestClient = AsynchronousJiraRestClientFactory()
         .createWithBasicHttpAuthentication(URI(config.host), config.username, config.password)
 ) {
+    /**
+     * Create a [comment] within the issue specified by the [issueKey].
+     */
+    @Suppress("UNUSED") // This is intended to be used via scripting.
+    fun createComment(issueKey: String, comment: String) {
+        val issue = restClient.issueClient.getIssue(issueKey).claim()
+
```

Comment:
Nit: Remove the blank line?

Reply:
`restClient.issueClient.addComment...` is the result of the method. To to be consistent with other code using an empty line before a `return` statement, I'd like to keep it as-is.